ConditionsService/data/Micheltbl.tbl

```diff
@@ -0,0 +1,5233 @@
+
+0.52	0.00006137198184
```

Comment:
The name here "Michel" - this is mu+ michel, does the name need to reflect that so it doesnt get confused with mu- DIO in the future

Reply:
I think it is okay as this is just the energy spectrum and is Completely charge symmetric. So Michel in this term could be used for both mu+ and mu- without trouble. One would only need to specific the pdgId in the fcl file.

Reply:
Yes, true I just worry people might think its the corrected on (Czernecki) but probably they wouldnt